src/Rx.ts

```diff
@@ -20,6 +20,7 @@ import './add/observable/fromArray';
 import './add/observable/fromEvent';
 import './add/observable/fromEventPattern';
 import './add/observable/fromPromise';
+import './add/observable/generate';
```

Comment:
@kwonoj kitechen sink imports all of Rx by default, so if it is part of rx, kitchen sink will see it too.